examples/helm/ha_cluster_with_proxy_and_password.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#
+# This values file overrides the default kube-starrocks chart to:
+# - add a default password for the root user
+# - deploy a highly-available StarRocks cluster with 3 FE pods and 3 BE pods
+# - deploy a LoadBalancer for the FE service
+# - deploy a LoadBalancer for the FE Proxy service
+# Note: This is not a production-level cluster, the CPU, Memory, and Disk resources
+# are set very low to allow this to be deployed in a Kubernetes cluster with limited
+# resources. For production sizing information please see:
+# https://docs.starrocks.io/docs/deployment/plan_cluster/
+#
+# To use this values file:
+# - [Add the Helm repo](../../doc/add_helm_repo_howto.md)
+# - Set a passord for the StarRocks database root user:
+#   kubectl create secret generic starrocks-root-pass --from-literal=password='g()()dpa$$word'
+# - Download this file:
+#   curl -O https://raw.githubusercontent.com/StarRocks/starrocks-kubernetes-operator/main/examples/examples/helm/ha_cluster_with_proxy_and_password.yaml
```

Comment:
should be better to rename the yaml file name, and append a `values` as its suffix